features/LdapManualImport.feature

```diff
@@ -0,0 +1,21 @@
+Feature: LDAP_Manual_Import
+    As a company administrator
+    I want to import manually users
+    In order to filter the ones who can access to Centreon Web
+	
+    Background:
+        Given I am logged in a Centreon server
+        And a LDAP configuration has been created 
+        And LDAP authentication is enabled
+        And users auto import is disabled
```

Comment:
Too much "And" : 
Given I am logged in a Centreon server
And a LDAP configuration has been created
And users auto import is disabled